.github/workflows/integration-specs.yml

```diff
@@ -0,0 +1,76 @@
+name: Integration Specs
+
+on: [push]
+
+jobs:
+  integration-specs:
+    runs-on: ubuntu-latest
+
+    services:
+      zookeeper:
+        image: confluentinc/cp-zookeeper
+        ports:
+          - 2181:2181
+
+      kafka:
+        image: confluentinc/cp-kafka
+        ports:
+          - 9092:9092
+          - 29092:29092
+        env:
+          KAFKA_BROKER_ID: 1
+          KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+          KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:29092,PLAINTEXT_HOST://localhost:9092
+          KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+          KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+          KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+
```

Comment:
Perhaps you need a health check after starting each service, before running the tests? You can pass arguments to [docker create](https://docs.docker.com/engine/reference/commandline/create/#options) using the [options](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservicesservice_idoptions) key – e.g.
```yaml
        options: --health-cmd "curl something --output /dev/null --silent --fail" --health-interval 10s --health-timeout 5s --health-retries 5
```

Reply:
Thanks! That did the trick! Took me a while to find a good health check command, though, but `kafka-topics` seems to do the trick and is bundled with the image :+1: